html/web3/cgi-bin/save_file.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python2
+
+import cgi, os, cgitb
+
+cgitb.enable()
+
+form = cgi.FieldStorage()
+
+# Get filename here.
+fileitem = form['filename']
+
+# Test if the file was uploaded
+if fileitem.filename:
+    # strip leading path from file name to avoid
+    # directory traversal attacks
+    fn = os.path.basename(fileitem.filename)
+    directory = '/tmp/webserv/'
```

Comment:
This may come from config?

Reply:
Good idea, done.

To test the CGI programs (until we implement CGI), we can use `python3` with the `http.server` module:
```
python3 -m http.server --cgi -b 127.0.0.1 8000
```

Reply:
I have no idea about how this works, but if it's a test, shouldn't it be in our CI system as a .github workflow?

Reply:
Sorry, I didn't see this comment. It might make sense to add a CI for testing when we implement CGI to our server. These web pages and CGI programs are a fast way for testing our server while implementing features. We might not include them in the final version.